context-monitoring/src/test/java/de/atb/context/TestCustomJsonFile.java

```diff
@@ -0,0 +1,39 @@
+package de.atb.context;
+
+import de.atb.context.common.util.ApplicationScenario;
+import de.atb.context.monitoring.models.CustomFileBasedDataModel;
+import de.atb.context.persistence.monitoring.MonitoringDataRepository;
+import de.atb.context.services.faults.ContextFault;
+import org.junit.Test;
+
+/**
+ * 
+ * 
+ * @author pvyas
+ * 
+ */
+public class TestCustomJsonFile {
+
+	@Test
+	public void doNothing() {
+		
+	}
+	
+	@SuppressWarnings("unchecked")
+	public static void main(String[] args) throws ContextFault {
```

Comment:
if I run this main method, I only get a warning:

`WARN [main] (MonitoringDataRepository.java:266) - No Monitoring Data with id 'ContextId_TankRefilling' found!`

Is this intentional? What is the purpose of this method?

Reply:
Sorry to create a confusion. Actually this file was copied for testing purpose but was not used. I have used test in `TestDataRetrieval.java`. So this custom test is not needed anymore. Hence file is deleted.